content/pages/comparisons/gitbook.html

```diff
@@ -0,0 +1,185 @@
+
+<html>
+  <head>
+    <title>Read the Docs vs. GitBook comparison</title>
+    <meta name="template" content="page" />
+    <meta name="slug" content="comparisons/gitbook" />
+    <meta name="status" content="hidden" />
+    <meta name="description" content="" />
+  </head>
+  <body>
+
+  {% import "partials/basic.html" as basic %}
+  {% import "partials/technical.html" as technical %}
+  {% import "partials/homepage.html" as homepage %}
+  {% import "partials/comparisons.html" as comparison %}
+
+  <section>
+    {% call basic.header_lead(
+    header="Read the Docs is the <em><strong>all-in-one</strong></em> solution you're looking for your documentation")
+    %}
+    <p>
+      GitBook is great. However, it lacks many features we consider <em>essentials</em> for technical teams.
+      When talking about technical documentation, <strong>Read the Docs</strong> is <em>the</em> perfect solution for it.
+    </p>
+    <p>
+      Let's compare it to know what you can expect when deploying your project's documentation to Read the Docs.
+    </p>
+    {% endcall %}
+  </section>
+
+
+<section id="compare">
+  <div class="ui padded container">
+
+    {% call comparison.feature_table(other_img="/images/gitbook-logo.svg", other_img_alt="GitBook logo") %}
+    {{ comparison.feature_row("Static hosting", "Publish any static content", "fa-page") }}
+    {{ comparison.feature_row("Build your site on the cloud", "Generate your site's content on the cloud", "fa-gears", on_other=True) }}
+    {{ comparison.feature_row("Deploy previews", "Solve style issues before deploying", "fa-rocket-launch", on_other=True) }}
+    {{ comparison.feature_row("Visual diff on deploy previews", "Visualize differences directly on the rendered version", "fa-face-monocle") }}
+    {{ comparison.feature_row("Search", "Always find what you are looking for", "fa-magnifying-glass", on_other=True) }}
+    {{ comparison.feature_row("Multiple versions", "Publish multiple versions of your documentation", "fa-books", on_other=True) }}
+    {{ comparison.feature_row("Redirects", "Redirect users to the right page", "fa-arrows-turn-right", on_other=True) }}
+    {{ comparison.feature_row("Custom 404 pages", "Style 404 pages with context-related information", "fa-xmark") }}
+    {{ comparison.feature_row("Custom \"robots.txt\"", "Improve your SEO by telling bots how to explore your site", "fa-robot") }}
+    {{ comparison.feature_row("Page analytics", "Understand how people browse your documentation", "fa-chart-mixed", on_other=True) }}
+    {{ comparison.feature_row("Search analytics", "Explore what people search for in your docs", "fa-chart-simple") }}
+    {{ comparison.feature_row("Translations", "Serve translated versions of your docs", "fa-language", on_other=True) }}
+    {{ comparison.feature_row("Private documention", "Give access only to trusted people", "fa-lock", on_other=True) }}
+    {{ comparison.feature_row("Open Source", "Maintained by the community", "fa-brands fa-osi") }}
```

Comment:
I think having a company behind RTD is important, so I think the lock in angle is stronger. 

```suggestion
    {{ comparison.feature_row("Open Source", "No lock in & contribute back", "fa-brands fa-osi") }}
```

Reply:
Yeap, this is the main thing (together with "you can use only one doctool") I want to highlight here.

This is one of the things I want to talk more with you to improve this message 😉